microsoft-edge/webview2/concepts/developer-guide.md

```diff
@@ -49,6 +49,9 @@ To take action when a new version is available, such as notifying the user to re
 ## Manage the lifetime of the user data folder 
 WebView2 apps create a user data folder to store data such as cookies, credentials, permissions, and so on. After creating the folder, your app is responsible for managing the lifetime of the user data folder, including clean up when the app is uninstalled.  For more information, navigate to [Managing the User Data Folder][Webview2ConceptsUserDataFolder].  
 
+## Process Failed
+WebView2 apps should listen for and handle the [ProcessFailed event][WebView2ProcessFailedEvent], as the browser process could fail at any time and should be exited or recreated gracefully.
```

Comment:
Might be worth explaining why it's important. Something like:

> WebView2 apps are supported by a collection of runtime processes that run alongside the app process. Although not expected, these processes might fail for a variety of reasons (for example, running out of memory, or being terminated by the user). When this happens, WebView2 notifies the application by raising the `ProcessFailed` event. WebView2 apps should listen for and handle this event so they can recover from these failures.

Reply:
updated, thx!